ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -728,3 +739,114 @@ def set_log_level(config):
     """
     level = config.getini("log_cli_level") or "INFO"
     log.setLevel(logging.getLevelName(level))
+
+
+global consumed_ram_start_test
+
+
+@pytest.hookimpl(tryfirst=True)
+def pytest_runtest_setup(item):
+    try:
+        start_monitor_memory()
+
+        global consumed_ram_start_test
+        consumed_memory = get_consumed_ram()
+        consumed_ram_start_test = get_consumed_ram()
+
+        log.debug(
+            f"Consumed memory at the start of TC {item.nodeid}: {bytes2human(consumed_memory)}"
+        )
+    except Exception:
+        log.warning(
+            f"Got exception while start monitor memory \n{traceback.format_exc()}"
+        )
+
+
+@pytest.hookimpl(trylast=True)
+def pytest_runtest_teardown(item):
+    try:
+        _, peak_rss_table, peak_vms_table = stop_monitor_memory(save_csv=False)
+        log.info(
+            f"Peak rss consumers during {item.nodeid}:\n"
+            f"{peak_rss_table.to_markdown(headers='keys', index=False, tablefmt='grid')}"
+        )
+        log.info(
+            f"Peak vms consumers during {item.nodeid}:\n"
+            f"{peak_vms_table.to_markdown(headers='keys', index=False, tablefmt='grid')}"
+        )
+
+        if ocsci_config.REPORTING.get("save_mem_report"):
```

Comment:
Please document this value in here: https://github.com/red-hat-storage/ocs-ci/blob/6ecc96ca640e96448a16e2271f5553a7a6248c12/conf/README.md#reporting

Reply:
done